test/typedecl_all/typedeclall.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+
+package typedeclall;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+
+option (gogoproto.testgen_all) = true;
+option (gogoproto.populate_all) = true;
+option (gogoproto.benchgen_all) = true;
+option (gogoproto.unmarshaler_all) = true;
```

Comment:
Interestingly this probably won't work as well when the marshaler and unmarshaler are not generated.  Perhaps we should make a note of this in the docs, that it is not supported without code generation